test/support/workflow_live_helpers.ex

```diff
@@ -201,10 +201,10 @@ defmodule Lightning.WorkflowLive.Helpers do
     |> render_hook("push-change", %{patches: patches})
   end
 
-  def add_job_patch(name \\ "") do
+  def add_job_patch(name \\ "", id \\ Ecto.UUID.generate()) do
```

Comment:
TIL that putting a function call inside a default _doesn't_ evaluate it on compilation. 

I totally expect that function to be called and all default values would be the same.